google-cloud/README.md

```diff
@@ -0,0 +1,49 @@
+# Kyma Google Cloud Marketplace
+
+## Overview
+
+This project provides Kyma configuration for Google Cloud Marketplace.
+
+## Prerequisites
+
+Use the following tools to set up the project:
+
+* [Docker](https://www.docker.com/)
+* [gcloud](https://www.docker.com/)
+* [envsubst](https://www.gnu.org/software/gettext/manual/html_node/envsubst-Invocation.html)
+
+## Usage
+
+### Install application on cluster manually
+
+To install application manually on Google Kubernetes Engine cluster, follow these steps:
```

Comment:
To install the application manually on a Google Kubernetes Engine cluster, follow these steps: